discovery_engine/lib/src/domain/models/search.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:hive/hive.dart'
+    show HiveType, HiveField, TypeAdapter, BinaryReader, BinaryWriter;
+import 'package:xayn_discovery_engine/src/domain/models/feed_market.dart';
```

Comment:
explicit show

Reply:
here I needed to expose generated type, but in general I wouldn't be so strict with explicit show in case of freezed classes